src/js/view/controls/rightclick.js

```diff
@@ -96,7 +96,7 @@ export default class RightClick {
         this.el.style.top = off.y + 'px';
         this.outCount = 0;
 
-        addClass(this.playerElement, 'jw-flag-rightclick-open');
+        addClass(this.player, 'jw-flag-rightclick-open');
```

Comment:
If this is still just the player element, let's continue to use `this.playerElement`. This makes it clear where you're adding the flag. `this.player` could mean a lot of things.

Reply:
`this.playerElement` please